.rubocop_todo.yml

```diff
@@ -59,6 +66,64 @@ Metrics/ParameterLists:
   Exclude:
     - 'dist/clouduploader.rb'
 
+# Offense count: 6
+RSpec/BeforeAfterAll:
+  Exclude:
+    - 'spec/spec_helper.rb'
+    - 'spec/rails_helper.rb'
```

Comment:
```suggestion
```
This line could be removed, if I'm not wrong. There is no `rails_helper.rb` file in `dist/t/spec/`.

Reply:
This is automatically generated with `rake dev:lint:rubocop:auto_gen_config:root`

Reply:
Hm. I guess it doesn't harm. LGTM (let's get this merged).